core-bundle/assets/styles/template-studio/operations.pcss

```
@@ -12,13 +12,11 @@
             background-image: url("../../../contao/themes/flexible/icons/delete.svg");
         }
 
-        &.operation--create_content_element_variant,
-        &.operation--create_frontend_module_variant {
+        &[class*="operation--create_"][class*="_variant"] {
```

```suggestion
        &[class*="operation--create_"] {
```

Less specific, we simply style it on the `create_` prefix (which will be different than `create` (and rename)

An idea would be adding css classes such as operation--icon-duplicate and operation--icon-edit but that would be out of scope for a bugfix.

---

It's for instance targeting the `CreateFooVariantOperation`,  `CreateBarVariantOperation` (and others). But not the `CreateBazHelperOperation`. The `_variant` bit should stay.